lib/go-atscfg/remapdotconfig_test.go

```diff
@@ -5132,11 +5132,11 @@ func TestMakeRemapDotConfigEdgeRangeRequestBGFetch(t *testing.T) {
 
 	txtLines := strings.Split(txt, "\n")
 
-	if len(txtLines) != 2 {
+	if len(txtLines) != 3 { // comment, blank, and remap
 		t.Fatalf("expected 1 remaps from HTTP_TO_HTTPS DS, actual: '%v' count %v", txt, len(txtLines))
```

Comment:
> "expected <del>1</del><ins>3</ins> remaps from..."

Reply:
There's only 1 remap, the other 2 lines are a comment and a blank